src/Symfony/Component/ErrorHandler/Resources/views/traces.html.php

```diff
@@ -1,21 +1,17 @@
 <div class="trace trace-as-html" id="trace-box-<?= $index; ?>">
     <div class="trace-details">
         <div class="trace-head">
-            <span class="sf-toggle" data-toggle-selector="#trace-html-<?= $index; ?>" data-toggle-initial="<?= $expand ? 'display' : ''; ?>">
-                <h3 class="trace-class">
-                    <span class="icon icon-close"><?= $this->include('assets/images/icon-minus-square-o.svg'); ?></span>
-                    <span class="icon icon-open"><?= $this->include('assets/images/icon-plus-square-o.svg'); ?></span>
-
-                    <span class="trace-namespace">
-                        <?= implode('\\', array_slice(explode('\\', $exception['class']), 0, -1)); ?><?= count(explode('\\', $exception['class'])) > 1 ? '\\' : ''; ?>
-                    </span>
-                    <?= ($parts = explode('\\', $exception['class'])) ? end($parts) : ''; ?>
-                </h3>
-
+            <div class="sf-toggle" data-toggle-selector="#trace-html-<?= $index; ?>" data-toggle-initial="<?= $expand ? 'display' : ''; ?>">
+                <span class="icon icon-close"><?= $this->include('assets/images/icon-minus-square-o.svg'); ?></span>
+                <span class="icon icon-open"><?= $this->include('assets/images/icon-plus-square-o.svg'); ?></span>
+                <h4 class="trace-namespace">
```

Comment:
This creates a `h4` being a header without any content. I don't think that's good.

IMO, the trace namespace and the trace class should both stay in the `<h3>`, so that the heading text is the FQCN.

Reply:
Fixed